src/core/index.ts

```diff
@@ -25,6 +24,14 @@ interface GetOptions {
   }) => void;
 }
 
+interface SetOptions {
```

Comment:
I think we should actually export these, that allows users of the library to use the typing.